scripts/preprocessing/04b-run_ssp.py

```diff
@@ -79,10 +83,15 @@ def run_ssp(*, cfg, subject, session=None):
     else:
         ch_names = None
 
-    reject_eog_ = config.get_ssp_reject(
-        ssp_type='eog',
-        epochs=(create_eog_epochs(raw)
-                if cfg.ssp_reject_eog == 'autoreject_global' else None))
+    reject_eog_ = None
+    eog_epochs = None
+    if cfg.ssp_reject_eog == 'autoreject_global':
+        eog_epochs = create_eog_epochs(raw)
+        if len(eog_epochs) >= 5:
+            reject_eog_ = config.get_ssp_reject(
+                ssp_type='eog',
+                epochs=eog_epochs)
+
     eog_projs, _ = compute_proj_eog(raw, ch_name=ch_names,
                                     average=cfg.eog_proj_from_average,
                                     reject=reject_eog_,
```

Comment:
I think this should now move inside a conditional and only get computed if enough events are preset.